foundry/game/File.py

```diff
@@ -39,16 +61,38 @@ def __init__(self, path: Optional[str] = None):
 
         self.position = 0
 
+    @staticmethod
+    def prg_normalize(offset: int) -> int:
+        """ Takes a vanilla ROM PRG offset and returns a
+        new offset that is correct for the current ROM's
+        PRG size
+        """
+        VANILLA_PRG_SIZE = 0x40000
+        # Only prg030 and prg031 are modified in an expanded ROM,
+        # so offsets to other banks should stay the same.
+        if offset < (BASE_OFFSET + (30 * 0x2000)):
+            return offset
+        # Otherwise, we need to normalize this bank 30 or 31
+        # offset to the last two banks based on PRG size
+        return ROM.header.prg_size - (VANILLA_PRG_SIZE-offset)
```

Comment:
Fix subtraction to follow PEP 8 guidelines
```suggestion
        return ROM.header.prg_size - (VANILLA_PRG_SIZE - offset)
```